### ISSUE TITLE
Vq-wav2vec finetuning

### ISSUE THREAD

**Issue by Sruestc** [pytorch/fairseq#2942]
I try to loaded pretrained model and finetuning the model on my own multi-lingual data.  
I found the parameters in the pretrained model are not the same as in scripts in examples/wav2vec  

After fixing the loss criterion (add binary_cross_entropy to criterion)
Then I found –combine-groups is False in the released model.
I fixed it then running the scripts. 
However the loss in random init is 8.9 ,while the loaded loss is 60 and the code ppl is 11 .
What is wrong??  Any other changes???
![image](https://user-images.githubusercontent.com/21210466/100055263-851ce900-2e5e-11eb-8dd4-ced2bcac219f.png)
![image](https://user-images.githubusercontent.com/21210466/100055281-8cdc8d80-2e5e-11eb-82b2-1747bc9c9eda.png)

**Comment by alexeib** [comment]
binary cross entropy criterion got renamed to "wav2vec" so you can use that as criterion

for one of the vq-wav2vec models i had to zero out the final prediction layer because the branch it was trained on had a different architecture that wasnt very trivial to migrate, but i think that was the kmeans one, not the gumbel version.

can you try to validate the vq-wav2vec model on librispeech devset and see what loss you get there?

**Comment by alexeib** [comment]
also since you are finetuning, you might wanna change --vq-temp as that is a function of num-updates and you are starting from 0 again
